cmd/zoekt-sourcegraph-indexserver/main_test.go

```diff
@@ -111,9 +113,14 @@ func TestListRepoIDs_Error(t *testing.T) {
 
 func TestMain(m *testing.M) {
 	flag.Parse()
+	level := sglog.LevelInfo
 	if !testing.Verbose() {
 		log.SetOutput(io.Discard)
+		level = sglog.LevelNone
 	}
+
+	logtest.InitWithLevel(m, level)
+	logger = sglog.Scoped("zoekt-sourcegraph-indexserver", "Zoekt-sourcegraph-indexserver logger for testing")
```

Comment:
My main comment would mean you don't need to do this here. We should be passing around a logger.

Reply:
I've removed the logger instance here. I still left the Initialization logic so that we can log messages during test runs in the future.